docs/en/sql-reference/sql-functions/dict-functions/dict_mapping.md

```diff
@@ -0,0 +1,215 @@
+
+- Optional parameters:
+  - `<value_column>`: The name of the value column, which is also the mapping column. If the value column is not specified, the default value column is the AUTO_INCREMENT column of the dictionary table.  The value column can also be defined as any column in the dictionary table excluding auto-incremented columns and primary keys. The column's data type has no restrictions.
+  - `<strict_mode>`: Whether to enable strict mode, that is, whether to return an error when the value mapped to the specified key is not found. If the parameter is set to `TRUE`,  an error is returned. If the parameter is set to `FALSE` (default),  `NULL` is returned.
+
+## Return Value
+
+The data type of the returned values remains consistent with the data type of the value column. If the value column is the auto-incremented column of the dictionary table, the data type of the returned values is BIGINT.
+
+However, when the value mapped to the specified key is not found, if the `strict_mode` parameter is set to `FALSE` (default),  `NULL` is returned. If the parameter is set to `TRUE`,  an error `ERROR 1064 (HY000): In strict mode, query failed if the record does not exist in the dict table` is returned.
+
+## Example
+
+**Example 1: Directly query the value mapped to a key from a dictionary table.**
+
+1. Create a dictionary table and load simulated data.
+
+      ```SQL
+      MySQL [test]> CREATE TABLE dict (
+          order_uuid STRING,
+          order_id_int BIGINT AUTO_INCREMENT 
+      )
+      PRIMARY KEY (order_uuid)
+      DISTRIBUTED BY HASH (order_uuid);
+      Query OK, 0 rows affected (0.02 sec)
+      
+      MySQL [test]> INSERT INTO dict (order_uuid) VALUES ('a1'), ('a2'), ('a3');
+      Query OK, 3 rows affected (0.12 sec)
+      {'label':'insert_9e60b0e4-89fa-11ee-a41f-b22a2c00f66b', 'status':'VISIBLE', 'txnId':'15029'}
+      
+      MySQL [test]> SELECT * FROM dict;
+      +------------+--------------+
+      | order_uuid | order_id_int |
+      +------------+--------------+
+      | a1         |            1 |
+      | a3         |            3 |
+      | a2         |            2 |
+      +------------+--------------+
+      3 rows in set (0.01 sec)
+      ```
+
+      > **NOTICE**
+      >
+      > Currently the `INSERT INTO` statement does not support partial updates. So please make sure that the values inserted into the `dict`'s key column are not duplicated. Otherwise, inserting the same key column value in the dictionary table multiple times causes its mapped value in the value column to change.
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/3f9d1da3-9d61-472a-80ce-b8f1a52456e3)
-> the `dict` dictionary table's